### PR TITLE
Apply Fork A/B Test Winner

### PR DESCRIPTION
**Pull request by officiallygod**
## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/bluehost/.github/blob/master/.github/contributing.md) doc
- [x] Linting and tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Changelog

Screenshot 2024-04-24 at 9 10 25 AM

### CODE REVIEW

File: src/OnboardingSPA/components/StartOptions/index.js [hunk 1]
```diff
@@ -1,34 +1,18 @@
-// WordPress
+import { SITEGEN_FLOW } from '../../data/flows/constants';
+import { resolveGetDataForFlow } from '../../data/flows';
 import { useSelect, useDispatch } from '@wordpress/data';
-import { memo, useEffect, useState } from '@wordpress/element';
```
Comment (arunshenoy99) [inline src/OnboardingSPA/components/StartOptions/index.js:3]: Let's group and classify the imports?

File: src/OnboardingSPA/steps/TheFork/contents.js [hunk 2]
```diff
@@ -10,21 +10,21 @@ const getContents = () => {
 		),
 		options: [
 			{
-				title: __( 'Guided Configuration', 'wp-module-onboarding' ),
+				title: __( ' Website Creator', 'wp-module-onboarding' ),
 				subtitle: __(
-					'Robust configuration guide to help you build your site',
+					'Unique AI generated content & design curated for you.',
```
Comment (arunshenoy99) [inline src/OnboardingSPA/steps/TheFork/contents.js:15]: Should we update the `.pot` file?

File: src/onboarding.js [hunk 3]
```diff
@@ -27,18 +24,19 @@ if ( runtimeDataExists ) {
...
+		// 	urls: {
+		// 		single: onboardingRestURL( 'events' ),
+		// 		batch: onboardingRestURL( 'events/batch' ),
+		// 	},
+		// 	settings: {
+		// 		debounce: {
+		// 			time: 3000,
+		// 		},
+		// 	},
+		// } );
```
Comment (arunshenoy99) [inline src/onboarding.js:39]: Let's remove

**Review by arunshenoy99 — COMMENTED** [review]
Should we change `Events::get_category()[0]` to `Events::get_category()` and make the latter a string like before in `includes/RestApi/EventsController.php`?

File: src/OnboardingSPA/components/StartOptions/index.js (hunk 1 above)
Comment (officiallygod) [reply src/OnboardingSPA/components/StartOptions/index.js:3]: Sure, Thanks.

File: src/onboarding.js (hunk 3 above)
Comment (officiallygod) [reply src/onboarding.js:39]: Sure

**Comment by officiallygod** [comment]
> Should we change `Events::get_category()[0]` to `Events::get_category()` and make the latter a string like before in `includes/RestApi/EventsController.php`?

Considering we might need to have an array in the long term as more keys could be there ( wonder-start, experiment ), due to this I left the code in place, as other's might not be aware of this little change.

File: src/OnboardingSPA/steps/TheFork/contents.js (hunk 2 above)
Comment (officiallygod) [reply src/OnboardingSPA/steps/TheFork/contents.js:15]: Should we do it now or with the release?

**Comment by arunshenoy99** [comment]
> > Should we change `Events::get_category()[0]` to `Events::get_category()` and make the latter a string like before in `includes/RestApi/EventsController.php`?
> 
> Considering we might need to have an array in the long term as more keys could be there ( wonder-start, experiment ), due to this I left the code in place, as other's might not be aware of this little change.

If you're not comfortable with removing it, you could write a validation callback that validates against the entire array instead of just the 0th element.

**Review by arunshenoy99 — APPROVED** [review]

**Review by arunshenoy99 — COMMENTED** [review]
QA done. Looks good!

**Pull request merged**